README.md

```diff
@@ -96,7 +97,22 @@ Connect to the container (with KUBECONFIG exported)
 make connect
 ```
 
-Accessing OKD UI
+To access the OKD UI from the host running `docker`, remember to export `OKD_CONSOLE_PORT=443` before `make cluster-up`.
+You should find out the IP address of the OKD docker container
 ```
-TODO - in the process of working out the details here.
+clusterip=$(docker inspect $(docker ps |grep quay.io/kubevirtci/ocp-4.3|awk '{print $1}')  | jq '.[0].NetworkSettings.IPAddress')
```

Comment:
maybe good to use grep KUBEVIRT_PROVIDER instead of 4.3
as in
https://github.com/kubevirt/kubevirtci/blob/5303219edbced17a28a70c71c7134235e63f781b/cluster-up/container.sh#L6
?

Reply:
ok